clients/admin-ui/src/features/dataset/DatasetFieldCell.tsx

```diff
@@ -29,20 +29,31 @@ const DatasetFieldCell = ({
   }
 
   if (attribute === "data_categories") {
-    const classifiedCategories = (classifyField?.classifications ?? []).map(
-      ({ label }) => label
+    const topClassification = classifyField?.classifications.reduce(
+      (maxClassification, next) => {
+        if (
+          (maxClassification.aggregated_score ?? 0) <
+          (next.aggregated_score ?? 0)
+        ) {
+          return next;
+        }
+        return maxClassification;
+      }
```

Comment:
Previously was just grabbing the first (top) classification but a failed Cypress test pointed me in the direction of copying the same functionality here to get the top 1 as used in https://github.com/ethyca/fides/blob/5a485387d8af247ec6479e4115088cbbb8394d77/clients/admin-ui/src/features/dataset/helpers.ts#L102 (there may be a better way for me to reuse this)

Reply:
hmm yeah, we may want to pull that `.reduce` out into its own func, also in `dataset/helpers.ts` and use it in both places?

Reply:
fixed! Thank you for the input 🙌🏽